### PR TITLE
Adding Color to list of endpoints.

### PR DESCRIPTION
**Pull request by hammygoonan**

### CODE REVIEW

File: clarifai/client/client.py [hunk 1]
```diff
@@ -739,3 +740,82 @@ def _base64_encoded_data_op(self, data, op):
     headers = self._get_json_headers()
     response = self._get_json_response(url, data=data, headers=headers)
     return self._parse_response(response)
+
+  def tag_color(self, files, model=None, local_ids=None, meta=None,
```
Comment (zeiler) [inline clarifai/client/client.py]: Maybe just color(). 

"model", "select_classes", and "language" are not supported so please remove from args and the docs below. You can juts pass in their defaults of None to the _multi_data_op call for now.

File: clarifai/client/client.py [hunk 2]
```diff
@@ -739,3 +740,82 @@ def _base64_encoded_data_op(self, data, op):
...
+                select_classes=None, language=None):
+    """ Autotag a single data file from an open file object or multiples data
```
Comment (zeiler) [inline clarifai/client/client.py]: Please update doc to mention color instead of tag.

File: clarifai/client/client.py [hunk 3]
```diff
@@ -739,3 +740,82 @@ def _base64_encoded_data_op(self, data, op):
...
+      clarifai_api = ClarifaiApi()
+      clarifai_api.tag_color([open('/path/to/local/image.jpeg'),
+                              open('/path/to/local/image2.jpeg')])
+    """
+    return self._multi_data_op(files, ['color'], model=model,
+                               local_ids=local_ids, meta=meta,
+                               select_classes=select_classes,
+                               language=language)
+
+  def tag_color_urls(self, urls, model=None, local_ids=None, meta=None,
```
Comment (zeiler) [inline clarifai/client/client.py]: Same fixes here as above.

**Comment by zeiler** [comment]
Thanks for the PR, just a few simple clean-ups of args and docs then we can merge!

**Comment by hammygoonan** [comment]
Hey @zeiler how's that now?

File: clarifai/client/client.py [hunk 4]
```diff
@@ -739,3 +740,75 @@ def _base64_encoded_data_op(self, data, op):
...
+      meta: a string of any extra information to accompany the request. This
+            has to be a string, so if passing structured data, pass a
+            json.dumps(meta) string.
+
+    Returns:
+      results: an API reponse including the generated tags. See the docs at
+      https://developer.clarifai.com/docs/ for more detais.
+
+    ## Example:
+    ```
```
Comment (robertwenquan) [inline clarifai/client/client.py]: This is a Markdown syntax, probably we don't want to keep it in the python code.

File: clarifai/client/client.py [hunk 5]
```diff
@@ -739,3 +740,75 @@ def _base64_encoded_data_op(self, data, op):
...
+                    Ex: "dog,cat,tree,car,boat"
+    language: set the default language using it's two letter (with options -XX
+              variant) ISO 639-1 code to use for all requests.
+
+    Returns:
+      results: an API reponse including the generated tags. See the docs at
+      https://developer.clarifai.com/docs/ for more detais.
+
+    ## Example:
+    ```
```
Comment (robertwenquan) [inline clarifai/client/client.py]: here as well

**Pull request merged**